app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -49,6 +51,16 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         revealInitialPrompt()
       })
 
+      this.$pageIsNotUsefulButton.on('click', function (e) {
+        var gaClientId
+        if (window.GOVUK.cookie('_ga') === null || window.GOVUK.cookie('_ga') === '') {
+          gaClientId = '111111111.1111111111'
```

Comment:
To make it clear what this is doing, it might be worth saving "111111111.1111111111" to a variable, e.g: dummyGAClientID.

Reply:
okay have done this and fixed a test that was failing. Hopefully builds will pass now.